packages/bridge/src/bridge.ts

```diff
@@ -39,8 +39,15 @@ const webBridge: WebBridge | undefined = IS_BROWSER_CLIENT
 export function createCondoBridge (): CondoBridge {
     const subscribers: Array<CondoBridgeSubscriptionListener> = []
 
+    /**
+     * Send params to container
```

Comment:
Wtf? What is a container? Why you’re adding this docs? It’s make no sense

Reply:
I'll remove comments to let you put proper description by yourself, since I'm not deep in purposes of bridge & UI packages